sklearn/tree/_splitter.pxd

```diff
@@ -75,7 +75,7 @@ cdef class Splitter:
 
     # Methods
     cdef int init(self, object X, const DOUBLE_t[:, ::1] y,
-                  DOUBLE_t* sample_weight) except -1
+                  const DOUBLE_t[:] sample_weight) except -1
```

Comment:
This also might be reformatted, but I can't suggest for some reasons.

Reply:
Done.